gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/GradleTestSpec.groovy

```diff
@@ -16,34 +16,45 @@
 
 package com.palantir.gradle.dist
 
-import com.energizedwork.spock.extensions.TempDirectory
 import groovy.transform.CompileStatic
 import groovy.transform.TypeCheckingMode
 import nebula.test.multiproject.MultiProjectIntegrationHelper
+import org.gradle.internal.impldep.com.google.common.collect.ImmutableList
```

Comment:
just-comment: in test where we control the classpath, less necessary to use a shaded version of a guava class